djoser/urls/base.py

```diff
@@ -1,10 +1,13 @@
 from django.conf.urls import url
+from djoser.conf import settings
 from django.contrib.auth import get_user_model
 
-from djoser import views
+from djoser import logging, views
 
 User = get_user_model()
 
+if settings.ENABLE_LOGGING :
+    logging.add_logging_mixin(views)
```

Comment:
anyway, i wonder if it's possible to do this from our goalie-backend? it would be more convenience for now since we are going to add loggin mixin to all of our other apps. We already use djoser and knox, and i suppose we are going to have another too soon.

Reply:
yeah its also possible @orhan89 , i proposed for us to create specific apps inside the project for our specific feature in our apps for example : `goalie-auth` to custom anything like inject the mixin, [something like my second option in my proposal](https://paper.dropbox.com/doc/Track-Authentication-rqLlIZqjcSsIFftORoAQM) ,  as our goalie-backend continue growing, it would be easy to manage

Reply:
if it's only to inject mixin like what you did here, i guess we can do that just from the project `goalie/urls.py` instead of creating a new app

Reply:
i just think for the future reference, for more complex situation like some complex feature in the future, or for small sample like you did to add new endpoint for login, we can did in specific apps though.